spec/models/course_spec.rb

```diff
@@ -846,4 +847,48 @@
       expect(course.pages_edited).not_to include(article2)
     end
   end
+
+  describe '#tracked_revisions' do
```

Comment:
I think this makes sense as its own file rather than added onto the already-huge course spec, especially since it's pretty independent of the rest of the specs here.